artifact/v3/scripts/executor.hpp

```diff
@@ -107,15 +107,15 @@ class ScriptRunner {
 	events::EventLoop &loop_;
 	bool is_artifact_script_;
 	chrono::seconds state_script_timeout_;
-	State state_;
-	Action action_;
 	string artifact_script_path_;
 	string rootfs_script_path_;
 	mender::common::processes::OutputCallback stdout_callback_;
 	mender::common::processes::OutputCallback stderr_callback_;
 	Error error_script_error_;
 	vector<string> collected_scripts_;
 	unique_ptr<mender::common::processes::Process> script_;
+	State state_;
+	Action action_;
```

Comment:
I think you meant to move these in the last commit, not in the first one :thinking: